### PR TITLE
feat: remove c as category url identifier

### PR DESCRIPTION
**Pull request by maxiroellplenty**
## Why:

Closes: #ID

## Checklist before requesting a review

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: apps/web/__tests__/test/feature/languageSelect.cy.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { paths } from '../../../utils/paths';
+
+const languageSelect = new LanguageSelectObject();
+
+beforeEach(() => {
+  cy.visitAndHydrate(paths.home);
+  cy.clearCookies();
+});
+
+describe('Smoke: Language Selector', () => {
```
Comment (ksted) [inline apps/web/__tests__/test/feature/languageSelect.cy.ts]:
```suggestion
describe('Feature: Language Selector', () => {
```

File: apps/web/__tests__/test/feature/languageSelect.cy.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  it('Check if Language selector works', () => {
```
Comment (ksted) [inline apps/web/__tests__/test/feature/languageSelect.cy.ts]:
```suggestion
  it('Should change the language from EN to DE', () => {
```

File: apps/web/__tests__/test/feature/languageSelect.cy.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    languageSelect.openModal();
+    languageSelect.checkIfModalIsOpen();
+    languageSelect.checkOptions();
+    languageSelect.selectOption('de');
+    languageSelect.checkLanguageSelected('de');
```
Comment (ksted) [inline apps/web/__tests__/test/feature/languageSelect.cy.ts:18]: **suggestion (non-blocking, we can change this later)**: Create a small domain-based abstraction layer.

File: apps/web/__tests__/test/feature/languageSelect.cy.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  });
+
+  it('Should change the category url after language switch', () => {
```
Comment (ksted) [inline apps/web/__tests__/test/feature/languageSelect.cy.ts]:
```suggestion
  it('Should stay on the same category page', () => {
```

File: apps/web/__tests__/test/feature/languageSelect.cy.ts [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    cy.visitAndHydrate('/living-room');
+    cy.intercept('/plentysystems/getFacet').as('getFacet');
+    cy.intercept('/plentysystems/getCategoryTree').as('getCategoryTree');
+
+    languageSelect.openModal().checkIfModalIsOpen().selectOption('de');
+    cy.wait(['@getFacet', '@getCategoryTree']);
+    cy.url().should('include', '/de/wohnzimmer');
+  });
+
+  it('Should change the product url after language switch', () => {
```
Comment (ksted) [inline apps/web/__tests__/test/feature/languageSelect.cy.ts]:
```suggestion
  it('Should stay on the same product page', () => {
```

File: apps/web/composables/useLocalization/useLocalization.ts [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+  };
+
+  /**
+   * @description Function for wrapping the product language path.
+   *
+   * @param path product path that is provided by the product response to redirect after a language switch
+   * @returns product path that is then navigated to
+   * @example buildProductLanguagePath('')
+   */
+  // eslint-disable-next-line sonarjs/no-identical-functions,unicorn/consistent-function-scoping
```
Comment (ksted) [inline apps/web/composables/useLocalization/useLocalization.ts:31]: **question**: Why can we not have a single `buildLocalizedPath` method and use it for both categories and products?

**Review by ksted — DISMISSED** [review]

**Review by ksted — DISMISSED** [review]

**Review by abocsan-plenty — APPROVED** [review]

**Pull request merged**